spk/hassio/src/service-setup.sh

```diff
@@ -0,0 +1,93 @@
+CFG_FILE="/usr/local/${SYNOPKG_PKGNAME}/etc/hassio.json"
+
+service_postinst() {
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+        DATA_DIR="${wizard_share_path}/${wizard_folder_name}"
+        # Fix install directory
+        mkdir -p "${DATA_DIR}"
+
+        HOMEASSISTANT_DOCKER="homeassistant/qemux86-64-homeassistant"
+        HASSIO_DOCKER="homeassistant/amd64-hassio-supervisor"
+
+        # Read infos from web
+        URL_VERSION="https://s3.amazonaws.com/hassio-version/stable.json"
+        HASSIO_VERSION=$(curl -s $URL_VERSION | jq -e -r '.supervisor')
+
+        # Pull supervisor image
+        /usr/local/bin/docker pull "${HASSIO_DOCKER}:${HASSIO_VERSION}" >/dev/null &&
+            /usr/local/bin/docker tag "${HASSIO_DOCKER}:${HASSIO_VERSION}" "${HASSIO_DOCKER}:latest" >/dev/null
+
+        if [ ! -f ${CFG_FILE} ]; then
+            mkdir -p /usr/local/${SYNOPKG_PKGNAME}/etc 
+            cp ${SYNOPKG_PKGDEST}/var/hassio.dist ${CFG_FILE}
+            # Write config
+            sed -i -e "s|@supervisor@|${HASSIO_DOCKER}|g" ${CFG_FILE}
+            sed -i -e "s|@homeassistant@|${HOMEASSISTANT_DOCKER}|g" ${CFG_FILE}
+            sed -i -e "s|@data_dir@|${DATA_DIR}|g" ${CFG_FILE}
+        fi
+    fi
+}
+
+service_preuninst() {
+    if [ ${SYNOPKG_PKG_STATUS} == "UNINSTALL" ]; then
+        HOMEASSISTANT="$(jq --raw-output '.homeassistant' ${CFG_FILE})"
+        SUPERVISOR="$(jq --raw-output '.supervisor' ${CFG_FILE})"
+        HASSIO_DATA="$(jq --raw-output '.data // "/usr/share/hassio"' ${CFG_FILE})"
+
+        docker rm --force homeassistant hassio_supervisor hassio_dns
```

Comment:
Perhaps make sure that `homeassistant` really is based on `homeassistant/qemux86-64-homeassistant`

Reply:
@chickenandpork no it is not satisfied, I'm not sure how to get that information from Docker..